tools/torch/data.lua

```diff
@@ -489,34 +674,33 @@ function DBSource:totalRecords ()
 end
 
 -- Derived class method close (LMDB flavour)
-function DBSource:lmdb_close (batchsize)
+function DBSource:lmdb_close (batchSize)
```

Comment:
not related to this patch but that makes me wonder why we have `batchSize` here... probably a left-over from a previous patch